scene/gui/progress_bar.cpp

```diff
@@ -85,20 +107,25 @@ void ProgressBar::_notification(int p_what) {
 	}
 }
 
-void ProgressBar::set_percent_visible(bool p_visible) {
-	percent_visible = p_visible;
+void ProgressBar::set_progress_label_format(ProgressLabelFormat p_type) {
+	progress_label_format = p_type;
 	update();
 }
 
-bool ProgressBar::is_percent_visible() const {
-	return percent_visible;
-}
+ProgressBar::ProgressLabelFormat ProgressBar::get_progress_label_format() const {
+	return progress_label_format;
+};
```

Comment:
I don't think this semicolon is necessary.

Reply:
These code ninjas leaving semicolons everywhere.

Thanks, removed it. :)